recipes-kernel/linux/files/patches-5.10/0121-scripts-dtc-Build-fdtoverlay-tool.patch

```diff
@@ -0,0 +1,51 @@
+From 2b14aa43f79131f4163fd3461aa0cdd5bf18e7db Mon Sep 17 00:00:00 2001
+From: Viresh Kumar <viresh.kumar@linaro.org>
+Date: Fri, 29 Jan 2021 12:54:06 +0530
+Subject: [PATCH 121/123] scripts: dtc: Build fdtoverlay tool
+
+commit 0da6bcd9fcc021ff2d6cd7c8173f42dcdff6a00b upstream.
+
+We will start building overlays for platforms soon in the kernel and
+would need fdtoverlay going forward. Lets start building it.
+
+The fdtoverlay program applies one or more overlay dtb blobs to a base
+dtb blob. The kernel build system would later use fdtoverlay to generate
+the overlaid blobs based on platform specific configurations.
+
+Signed-off-by: Viresh Kumar <viresh.kumar@linaro.org>
+Signed-off-by: Rob Herring <robh@kernel.org>
+Link: https://lore.kernel.org/r/4a201dea3ba11a00cab7e936dfc1140dac1a1ae3.1611904394.git.viresh.kumar@linaro.org
+[s-anna@ti.com: cherry-pick commit '0da6bcd9fcc0' from v5.12]
+Signed-off-by: Suman Anna <s-anna@ti.com>
```

Comment:
What is the upstream status of this? I don't want to start carrying stuff that has no perspective.

Reply:
OK, this is https://github.com/torvalds/linux/commit/0da6bcd9fcc021ff2d6cd7c8173f42dcdff6a00b. Please always pick backports from upstream, never from TI.

Reply:
But aren't we missing https://github.com/torvalds/linux/commit/b775f49fbc8b07b0c3a4079e6f09f192eaf6e92f?

Reply:
> But aren't we missing [torvalds/linux@b775f49](https://github.com/torvalds/linux/commit/b775f49fbc8b07b0c3a4079e6f09f192eaf6e92f)?
Doesn't need this patch and seems to work